recipe/meta.yaml

```diff
@@ -35,8 +35,10 @@ requirements:
     - zstd
     # see: https://github.com/conda-forge/google-cloud-cpp-feedstock/pull/108,
     - libcrc32c  # [gcs == 'gcs_enabled' and win]
-    - libcurl    # [gcs == 'gcs_enabled' and win]
     - libxml2
+  run:
+    - libcurl    # [win]
```

Comment:
Please move the libcurl requirement back to the `host` section. It should still be installed at runtime due to it's [`run_exports`](https://github.com/conda-forge/curl-feedstock/blob/5001d31c3fa114830fa61f22bc97dc8e3298d94e/recipe/meta.yaml#L49)

Reply:
I'm currently writing a treatise length Issue on this. I'll share when ready

Reply:
@Shelnutt2 As promised: https://github.com/conda-forge/tiledb-feedstock/issues/224